insta-unfollower.py

```diff
@@ -90,6 +92,15 @@ def get_user_profile(username):
     response = json.loads(extract.group(1))
     return response['entry_data']['ProfilePage'][0]['graphql']['user']
 
+def get_allow_list():
+    allow_list = []
+    with open('allowlist.csv') as csv_file:
```

Comment:
Declare the filename in a var at the top of the file  after cache and route vars